Content.Client/Power/PowerMonitoringWindow.xaml.Widgets.cs

```diff
@@ -478,6 +505,31 @@ public PowerMonitoringButton()
 
         MainContainer.AddChild(NameLocalized);
 
+        BatteryLevel = new ProgressBar()
+        {
+            HorizontalAlignment = HAlignment.Right,
+            SetWidth = 60f,
+            Margin = new Thickness(15, 7, 0, 7),
+            MinValue = 0,
+            MaxValue = 100,
+            Page = 0,
+            Value = 0,
```

Comment:
You're just setting all of these to the default, there's no point to.

Also set `MaxValue` to 1 so you can avoid multiplying by 100 when setting the value.